.github/workflows/k8s-ci.yml

```diff
@@ -210,10 +210,9 @@ jobs:
           - test-target: pre_merge
             range: ""
             values_file: tests/deploy/values/ci.yml
-          # Skipping until https://github.com/elastic/security-team/issues/6757 is resolved
-          # - test-target: aws
-          #   range: ""
-          #   values_file: tests/deploy/values/ci-aws-services.yml
+          - test-target: 'aws and not aws_s3'
```

Comment:
Implemented skip for AWS S3 functional tests.
A new [task](https://github.com/elastic/security-team/issues/7282) has been created.

Reply:
Can you elaborate what's the meaning of the value `aws and not aws_s3`?

Reply:
We run tests using markers (-m) or keyword expressions (-k). 
When using markers, our command looks like `pytest -m "pre_merge"`. 

With keyword expressions, we can define patterns to select specific tests for execution. The value `aws and not aws_s3` is a way to filter tests based on their names. The expression `aws and not aws_s3` is used to execute all tests whose names contain the term `aws` but do not contain the term `aws_s3`.

Reply:
So if I understand correct, all the values under `test-target` variable are pytest markers?

Reply:
> So if I understand correct, all the values under `test-target` variable are pytest markers?

exactly